src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,24 +14,399 @@ tree_item_index: 1100
 
 {{! excerpt}}
 The [Nuxeo Aspera](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-aspera) connector
-enables users to download/upload binaries with Aspera.
+enables users to download/upload binaries with Aspera.  It is designed to be used with **Aspera On Cloud** instances, however, if you use a **self-hosted Aspera** instance, it is still possible you will be able to use the Nuxeo Aspera Connector.  Contact your Aspera team for more information and assistance.
```

Comment:
I am wondering if we should or not :) add the Aspera link documentation to `Aspera On Cloud`

Reply:
@RSalem07 good idea.  would help us really "hit home" about the fact this is geared to AOC.